warehouse/validations/validate.go

```diff
@@ -527,7 +527,10 @@ func createManager(ctx context.Context, dest *backendconfig.DestinationT) (manag
 		return nil, fmt.Errorf("getting manager: %w", err)
 	}
 
-	operations.SetConnectionTimeout(warehouseutils.TestConnectionTimeout)
+	operations.SetConnectionTimeout(warehouseutils.GetConnectionTimeout(
+		destType,
+		dest.ID,
+	))
```

Comment:
We should probably keep this as it is since it is used for destination validations from the control plane.